ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/shared-phz/deprovision/cucushift-installer-rehearse-aws-ipi-shared-phz-deprovision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-shared-phz-deprovision
+  steps:
+  - chain: cucushift-installer-rehearse-aws-ipi-byo-route53-deprovision
+  env:
+  - name: ENABLE_SHARED_PHZ
```

Comment:
If this chain, just set `ENABLE_SHARED_PHZ` with a different value on `cucushift-installer-rehearse-aws-ipi-byo-route53-deprovision`, I think we can directly use it in install-workflow.

Reply:
Updated